Frontend/app/types/Announcement.ts

```diff
@@ -0,0 +1,7 @@
+export type Announcement = {
+  id: number;
+  announcer: string;
+  title: string;
+  date: Date;
+  announcement: string;
+};
```

Comment:
Deze shit komt uiteindelijk uit de backend toch? Dan kan je beter eerst even de backend afbouwen zodat we het via heyapi kunnen genereren